Upstream-Status: Pending

`B' in file 4.x has the same meaning with `W' in file 5,
let `B' be an alias of `W'.

8/17/2010 - created by Qing He <qing.he@intel.com>

diff --git a/src/apprentice.c b/src/apprentice.c
index 58826c3..3d4c3cf 100644
--- a/src/apprentice.c
+++ b/src/apprentice.c
@@ -1340,6 +1340,7 @@ parse(struct magic_set *ms, struct magic_entry **mentryp, uint32_t *nmentryp,
 						    "zero range");
 					l = t - 1;
 					break;
+				case CHAR_COMPACT_BLANK:
 				case CHAR_COMPACT_WHITESPACE:
 					m->str_flags |= STRING_COMPACT_WHITESPACE;
 					break;
diff --git a/src/file.h b/src/file.h
index c07f2d4..42cf416 100644
--- a/src/file.h
+++ b/src/file.h
@@ -282,6 +282,7 @@ struct magic {
 #define STRING_TEXTTEST				BIT(5)
 #define STRING_BINTEST				BIT(6)
 #define CHAR_COMPACT_WHITESPACE			'W'
+#define CHAR_COMPACT_BLANK			'B'
 #define CHAR_COMPACT_OPTIONAL_WHITESPACE	'w'
 #define CHAR_IGNORE_LOWERCASE			'c'
 #define CHAR_IGNORE_UPPERCASE			'C'
